k8s/deployments/mariadb.yaml

```diff
@@ -33,26 +21,14 @@ spec:
                   name: mpm-secrets
                   key: DB_PASSWORD
             - name: MYSQL_DATABASE
-              value: micro_power_manager
+              value: database-1
```

Comment:
Please leave the database name `micro_power_manager` It is hardcoded in some other places. This will probably require provisioning a database in the cluster.